File: POOJAVA/src/clasesPOO/Persona.java
package clasesPOO;

public class Persona 
{
    int idPers;
    String idTipoDoc;
    String NDoc;
    String apellidos;
    String nombres;
    int edad;
    String genero;
    String direccion;

    public Persona(String nomb,String apell)
    {
        establecerDatos(nomb,apell);
    }
    public String ObtenerNombres()
    {
       return nombres;
    }
    public String ObtenerApellidos()
    {
       return apellidos;
    }
    void  establecerDatos(String nom,String ape)
    {
        nombres=nom;
        apellidos=ape;
    }
}

File: POOPHP/app/Model/Persona.php
<?php
namespace App\Model;
class persona
{
    public $idPers="";
    public $idTipoDoc="";
    public $NDoc="";
    public $apellidos="";
    public $nombres="";
    public $edad="";
    public $genero="";
    public $direccion="";

   public function __construct(String $nomb,String $apell)
    {
        $this->establecerDatos($nomb,$apell);
    }
    public function ObtenerNombres()
    {
        return $this->nombres;
    }
    public function ObtenerApellidos()
    {
        return $this->apellidos;
    }
    public function establecerDatos(String $nom,String $ape)
    {
        $this->nombres=$nom;
        $this->apellidos=$ape;
    }
}
?>